### PR TITLE
Remove "Runtime" from all namespaces.

### LINKED ISSUE #1697: Remove "Runtime" from all namespace reference, sort namespaces as per .net convention and enforce the same.
Remove "Runtime" from all namespace reference, sort namespaces as per .net convention listed below and enforce the same.

**SA1210:** Using directives must be ordered alphabetically by namespace.
**SA1208:** System using directives must be placed before other using directives.

### PR DESCRIPTION
**Pull request by codemzs**
Partially fixes #1697 by removing "Runtime" from all namespaces and that is it, no moving classes to other namespaces.

### CODE REVIEW

**Comment by TomFinley** [comment]
@codemzs out of curiosity what is your plan w.r.t. namespace sorting and the like? Not do it, other?

**Comment by codemzs** [comment]
@TomFinley I plan to automate that using a script in a separate change but if you insist we do it in this change that is also fine by me but please let me know.

**Comment by TomFinley** [comment]
No, I do not insist, just wanted to know plan is all, thanks.

File: docs/samples/Microsoft.ML.Samples/Dynamic/IidChangePointDetectorTransform.cs [hunk 1]
```diff
@@ -1,13 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
```
Comment (TomFinley) [inline docs/samples/Microsoft.ML.Samples/Dynamic/IidChangePointDetectorTransform.cs]: Interesting, good one, I wonder if we should have some mechanism to check for this sort of thing, right now it's purely manual.

File: src/Microsoft.ML.EntryPoints/ImportTextData.cs [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
 
 using Microsoft.ML.Data;
-using Microsoft.ML.Runtime;
-using Microsoft.ML.Runtime.CommandLine;
-using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML;
+using Microsoft.ML.CommandLine;
+using Microsoft.ML.EntryPoints;
+using System;
```
Comment (justinormont) [inline src/Microsoft.ML.EntryPoints/ImportTextData.cs]: Know why System got added?

Comment (codemzs) [reply src/Microsoft.ML.EntryPoints/ImportTextData.cs]: Probably due to a dependency. The next PR on this effort will sort the namespaces and remove unused namespaces from the entire codebase so I wouldn’t worry about potential unused namespace.

File: src/Microsoft.ML.EntryPoints/ModelOperations.cs [hunk 3]
```diff
@@ -2,16 +2,17 @@
...
-using Microsoft.ML.Runtime;
-using Microsoft.ML.Runtime.CommandLine;
-using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML;
+using Microsoft.ML.CommandLine;
+using Microsoft.ML.Data;
+using Microsoft.ML.EntryPoints;
+using Microsoft.ML.Learners;
 using Microsoft.ML.Trainers;
```
Comment (justinormont) [inline src/Microsoft.ML.EntryPoints/ModelOperations.cs]: Seems odd that we'd have `Microsoft.ML.Trainers` & `Microsoft.ML.Learners`, given the similar naming.

Also, it's not clear to me why `Microsoft.ML.Learners` was added to the list.

Was `Trainers` partially renamed to `Learners` and now we have both? _(rampant speculation)_

Comment (codemzs) [reply src/Microsoft.ML.EntryPoints/ModelOperations.cs]: This change does not rename Trainers to Learners. It just removes runtime from all namespaces.

**Comment by justinormont** [comment]
I don't think there's a better way for this PR (hard to slice this PR in to parts), but w/ 923 files changed, the GitHub web GUI is pretty inadequate for the task. 

If anyone is tired of opening each "Load diff" for each file, you can put this your browser's JS console to open all: 
```javascript
[...document.getElementsByTagName('button')].filter(a=>a.innerText.trim() === 'Load diff').forEach(a=>a.click())
```

One less tedious step :)

File: src/Microsoft.ML.Legacy/Runtime/Internal/Tools/CSharpGeneratorUtils.cs [hunk 4]
```diff
@@ -296,7 +296,7 @@ private static string GetCharAsString(char value)
                     if (generatedClasses.IsGenerated(fieldType.FullName))
                         return generatedClasses.GetApiName(fieldType, rootNameSpace) + "." + enumAsString;
                     else
-                        return generatedClasses.GetApiName(fieldType, "Runtime") + "." + enumAsString;
+                        return generatedClasses.GetApiName(fieldType, "") + "." + enumAsString;
```
Comment (justinormont) [inline src/Microsoft.ML.Legacy/Runtime/Internal/Tools/CSharpGeneratorUtils.cs]: fyi -- so anyone else reading this doesn't go and check, `GetApiName()` seems to handle `""`:
https://github.com/dotnet/machinelearning/blob/a8b844ccee21b7639e1b0c24fefd1e69088f0d7e/src/Microsoft.ML.Legacy/Runtime/Internal/Tools/GeneratedClasses.cs#L27-L38

File: src/Microsoft.ML.TimeSeries/TimeSeriesProcessing.cs [hunk 5]
```diff
@@ -2,17 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.ML.Runtime.EntryPoints;
-using Microsoft.ML.Runtime.TimeSeriesProcessing;
+using Microsoft.ML.EntryPoints;
+using Microsoft.ML.TimeSeriesProcessing;
 
-[assembly: EntryPointModule(typeof(TimeSeriesProcessing))]
+[assembly: EntryPointModule(typeof(TimeSeriesProcessingEntryPoints))]
```
Comment (justinormont) [inline src/Microsoft.ML.TimeSeries/TimeSeriesProcessing.cs]: @ganik, @montebhoover: I assume NimbusML isn't taking in the TimesSeries work yet. If so, the renaming of the EntryPoint might affect NimbusML.

**Comment by codemzs** [comment]
@justinormont Good idea, will do that. Thanks for reviewing!
@TomFinley thanks for reviewing.

**Pull request merged**